Makefile

```diff
@@ -45,7 +45,7 @@ $(LOCAL_BINARY): $(SOURCES) GITCOMMIT_SHA
 
 .PHONY: test
 test:
-	. ./scripts/shared_env && go test -v -timeout 30s -short -cover $(shell go list ./ecr-login/... | grep -v /vendor/)
```

Comment:
Are vendored dependencies excluded now?

Reply:
Yes. Seems Go can recognize the vendored dependencies as their original names, not `github.com/awslabs/amazon-ecr-credential-helper/ecr-login/vendor/github.com/...`. So it doesn't match `./...`.

* Go 1.11: https://github.com/awslabs/amazon-ecr-credential-helper/pull/244/checks?check_run_id=1393304202
* Go 1.15: https://github.com/awslabs/amazon-ecr-credential-helper/pull/244/checks?check_run_id=1393304305